themes/default/layouts/shortcodes/configure-gcp.html

```diff
@@ -3,5 +3,5 @@
 `config` command using the project's ID: ```bash gcloud config set project
 <YOUR_GCP_PROJECT_ID>
     ``` Next, Pulumi requires default application credentials to interact with your Google Cloud resources, so run `auth application-default login` command to obtain those
-    credentials. ```bash gcloud auth application-default login ```</YOUR_GCP_PROJECT_ID
+    credentials: ```bash $ gcloud auth application-default login ```</YOUR_GCP_PROJECT_ID
```

Comment:
```suggestion
    credentials: ```bash $ gcloud auth application-default login ```
```